firstvoices/backend/tasks/mtd_export_tasks.py

```diff
@@ -186,38 +206,57 @@ def check_sites_for_mtd_sync(self):
     logger.info(ASYNC_TASK_START_TEMPLATE)
 
     try:
-        self.state = "STARTED"
         sites = Site.objects.filter(visibility=constants.Visibility.PUBLIC)
-        six_hours_ago = timezone.now() - timedelta(hours=6)
 
         for site in sites:
-            updated_entries_count = DictionaryEntry.objects.filter(
-                site=site, last_modified__gte=six_hours_ago
-            ).count()
-
-            updated_categories_count = DictionaryEntryCategory.objects.filter(
-                category__site=site, last_modified__gte=six_hours_ago
-            ).count()
-
-            updated_related_media_count = (
-                DictionaryEntry.objects.filter(site=site)
-                .filter(
-                    Q(related_audio__last_modified__gte=six_hours_ago)
-                    | Q(related_images__last_modified__gte=six_hours_ago)
-                    | Q(related_videos__last_modified__gte=six_hours_ago)
-                )
-                .distinct()
-                .count()
+            completed_jobs = MTDExportJob.objects.filter(
+                site=site, status=JobStatus.COMPLETE
             )
+            if completed_jobs.exists():
+                last_export = completed_jobs.latest()
+                last_modified_check_period = last_export.created
```

Comment:
Super minor, but could we please call this variable last_export_created ? I got confused because it is a timestamp not a period